tests/e2e/Services/Users/UsersBase.php

```diff
@@ -1038,29 +1038,65 @@ public function testUpdateUserPassword(array $data): array
     /**
      * @depends testGetUser
      */
-    #[Retry(count: 1)]
     public function testUpdateUserStatus(array $data): array
     {
-        /**
-         * Test for SUCCESS
-         */
-        $user = $this->client->call(Client::METHOD_PATCH, '/users/' . $data['userId'] . '/status', array_merge([
+        // Verify that the user exists
+        $response = $this->client->call(Client::METHOD_GET, '/users/' . $data['userId'], array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()));
+
+        $this->assertEquals($response['headers']['status-code'], 200);
+
+        // Test: Activate user
+        $response = $this->client->call(Client::METHOD_PATCH, '/users/' . $data['userId'] . '/status', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'status' => true
```

Comment:
No need to do this because they are already active and the end of the test tests for activating.

Reply:
This hasn't been addressed